apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -270,18 +270,16 @@ def count(self) -> int:
             Experimental.
         """
         self._check_open_read()
-        return cast(int, self._soma_reader().nnz())
+        handle: clib.DataFrameWrapper = self._handle
+        return cast(int, handle.count)
 
-    def enumeration(self, name: str) -> Tuple[Any, ...]:
-        """Doc place holder.
-
-        Returns:
-            Tuple[Any, ...]: _description_
-        """
-        return tuple(self._soma_reader().get_enum(name))
-
-    def column_to_enumeration(self, name: str) -> str:
-        return str(self._soma_reader().get_enum_label_on_attr(name))
+    def enumeration(self, name: str) -> Optional[Tuple[Any, ...]]:
```

Comment:
To avoid use of `Any`, it may be better to call this `Tuple[object, ...]`. Doing so would have the advantage that the caller would need to either check or assert that it gets the returned type it wants, which is safer. But also it has the disadvantage that the caller would need to either check or assert that it gets the returned type it wants, which is kind of annoying.